Makefile

```diff
@@ -55,6 +55,9 @@ clean: ## Clean workspace
 	@ $(MAKE) --no-print-directory log-$@
 	rm -rf ./$(BUILD_DIR)
 
+.PHONY: deps
```

Comment:
Nice catch, but can you rename the `deps` in `all` target to `vendor` instead of creating new one?

Reply:
You mentioned this target is being used in an external scripts and Docker image. Can you link any of those for reference here?